### PR TITLE
Some additions to PropertyDefinitons and RefdataValues

### PR DESCRIPTION
**Pull request by darupp**

### CODE REVIEW

File: app/web-app/setup/PropertyDefinition.csv [hunk 1]
```diff
@@ -68,7 +68,7 @@ License Property,Indemnification by licensee clause indicator,Entschädigung dur
...
 License Property,Invoicing,Rechnungsstellung Datum,Invoicing Date,class java.util.Date,,,,,Datum der Rechnungsstellung.,,,,,
-License Property,Invoicing Consortial,Rechnungsstellung konsortial,Invoicing consortial,class com.k_int.kbplus.RefdataValue,invoicing,,,,Benennung des Rechnungsstellers (Anbieter oder Konsortium),Defining who creates and sends out invoices to the participants.,,,,
+License Property,Invoicing Consortial,Rechnungsstellung konsortial,Invoicing consortial,class com.k_int.kbplus.RefdataValue,invoicing,,,,Benennung des Rechnungsstellers (Anbieter, Konsortium, Zwischenhändler etc.),Defining who creates and sends out invoices to the participants.,,,,
```
Comment (croqueGrec09) [inline app/web-app/setup/PropertyDefinition.csv:71]: Zwischenhändler, etc.

Comment (klodav) [inline app/web-app/setup/PropertyDefinition.csv:71]: :warning: --> use quotechar ` " ` if string contains separator ` , `

Comment (darupp) [reply app/web-app/setup/PropertyDefinition.csv]: Added quotechars

Comment (darupp) [reply app/web-app/setup/PropertyDefinition.csv]: Added comma

**Pull request merged**